package.json

```diff
@@ -11,6 +11,7 @@
   "devDependencies": {
     "@babel/cli": "^7.4.4",
     "@babel/core": "^7.4.4",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
```

Comment:
Kinda breaking my own rule here in terms of not adding things that are not standardised…but node.js and chrome already ship implementations

Reply:
We are we removing `object-rest-spread`?